CppCoreGuidelines.md

```diff
@@ -16867,6 +16867,47 @@ Static helps dynamic: Use static polymorphism to implement dynamically polymorph
 Dynamic helps static: Offer a generic, comfortable, statically bound interface, but internally dispatch dynamically, so you offer a uniform object layout.
 Examples include type erasure as with `std::shared_ptr`'s deleter (but [don't overuse type erasure](#Rt-erasure)).
 
+    #include <memory>
+
+    class Object {
+    public:
+        template <typename T>
+        Object(T && obj)
+            : concept_(std::make_shared<Model<T>>(std::forward<T>(obj))) {}
+
+        int get_id() const { return concept_->get_id(); }
+
+    private:
+        struct Concept {
+            virtual ~Concept() {}
+            virtual int get_id() const = 0;
+        };
+
+        template <typename T>
```

Comment:
I think this line has a more space between `template <typename T>` compared to other examples